client/js/screens/LogIn/styles.js

```diff
@@ -2,8 +2,42 @@ import { StyleSheet } from "react-native";
 import { globalStyles } from "../../../config/styles";
 
 const styles = StyleSheet.create({
-  container: globalStyles.container
+  container: globalStyles.container,
```

Comment:
use Dimensions / resizemode for height, width. Use flexbox to avoid having huge margins/paddings like 50.